projects/client/RabbitMQ.Client/src/util/BlockingCell.cs

```diff
@@ -41,6 +41,8 @@
 using System;
 using System.Threading;
 
+#pragma warning disable CS0649
```

Comment:
Hmm, this is interesting. What problem are you running into here?

Reply:
I was getting a build warning but now I'm not. Bizarre.

Reply:
It was a warning about [this line](https://github.com/rabbitmq/rabbitmq-dotnet-client/blob/master/projects/client/RabbitMQ.Client/src/util/BlockingCell.cs#L60) being unassigned and thus keeping its default value.